pkg/logproto/logproto.proto

```diff
@@ -67,10 +66,9 @@ message Entry {
 
 message TailRequest {
   string query = 1;
-  string regex = 2;
-  uint32 delayFor = 3;
-  uint32 limit = 4;
-  google.protobuf.Timestamp start = 5 [(gogoproto.stdtime) = true, (gogoproto.nullable) = false];
```

Comment:
I think this is going to break old clients (i.e., during rollout), we should just add `reserved 2;` and keep the other field numbers the same.